cmd/pg/backup_fetch.go

```diff
@@ -12,25 +13,36 @@ const (
 matches given shell file pattern.
 For information about pattern syntax view: https://golang.org/pkg/path/filepath/#Match`
 	RestoreSpecDescription = "Path to file containing tablespace restore specification"
+	ReverseDeltaUnpackDescription = "Unpack delta backups in reverse order (testing)"
```

Comment:
testing -> beta feature

Reply:
Fixed in 1659a812fdd448d376f9947fcc579956e006e151